fabfile.py

```diff
@@ -1662,6 +1662,20 @@ def docker_compose():
     # run("docker-compose -f docker/build/docker-compose.yml up")
 
 
+@task
+def get_ssl_certificates():
+    if env.oscp_path:
+        root_certificate = run('curl https://letsencrypt.org/certs/isrgrootx1.pem.txt')
+        intermediate_certificate_1 = run('curl https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem.txt')
+        intermediate_certificate_2 = run('curl https://letsencrypt.org/certs/letsencryptauthorityx3.pem.txt')
+        with open(env.oscp_path, 'w') as certificates_file:
+            for certificate_file in [root_certificate, intermediate_certificate_1, intermediate_certificate_2]:
```

Comment:
Put these elements in a tuple instead of a list. In this case, completely insignificant. But for your future knowledge:
https://stackoverflow.com/questions/68630/are-tuples-more-efficient-than-lists-in-python